src/store/ChartStore.js

```diff
@@ -446,6 +429,7 @@ class ChartStore {
     setMainSeriesDisplay(name) {
         // Set display name of main series (to be shown in crosshair tooltip)
         this.stxx.chart.seriesRenderers._main_series.seriesParams[0].display = name;
+        // TODO
```

Comment:
@shayanbinary what's this.

Reply:
@brucebinary :D a todo so that we remember, we should change this line of code